dockerfile/ci-tasks/github-action-run-once

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+# Register a self-hosted runner on GitHub, wait for and run one request, and stop again.
+# This ensures that broken/malicious PRs cannot leave anything permanent behind on the host machine.
+# Inputs:
+#   $GITHUB_TOKEN: must have "public_repo" scope
+#   $GITHUB_REPOSITORY: owner/name
+#   $RUNNER_NAME: string, optional (defaults to container host name)
+#
+# Example invocation:
+# podman run -h local-runner-1 --rm -e GITHUB_TOKEN=$(< ~/.config/github-token) -e GITHUB_REPOSITORY=rhinstaller/anaconda anaconda/ci-tasks github-action-run-once
+#
+# Documentation:
+# https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/about-self-hosted-runners
+# https://developer.github.com/v3/actions/self-hosted-runners/
+
+set -eu
+
+export RUNNER_ALLOW_RUNASROOT=1
+cd /actions-runner
+
+# get the runner registration token
+RUNNER_TOKEN=$(curl --silent --show-error -X POST -u "token:$GITHUB_TOKEN" \
+    "https://api.github.com/repos/$GITHUB_REPOSITORY/actions/runners/registration-token" | \
+    sed -n '/"token"/ { s/.*: "//; s/".*$//; p }')
+if [ -z "$RUNNER_TOKEN" ]; then
+    echo "Failed to acquire runner token" >&2
+    exit 1
+fi
+
+# register runner
+./config.sh --labels 'ci-tasks' ${RUNNER_NAME:+--name} ${RUNNER_NAME:-} --replace --token "$RUNNER_TOKEN" --unattended --url "https://github.com/$GITHUB_REPOSITORY"
```

Comment:
This is a bit crazy bashism to me but I was able to understand what is happening there.

Reply:
It's not a bashism, it's POSIX shell. That said, if you think it's weird, I'm happy to replace with an if condition and building some `$EXTRA_ARGS` or so. (That's why we have reviews, thanks for pointing out!)

Reply:
This one seems self-explanatory to me, and I have not been exposed to shells too much... Then again maybe trusting that hunch is not such a great idea. It's shell after all...